layouts/partials/header.html

```diff
@@ -27,6 +27,7 @@
     <meta name="keywords" content="{{ .Keywords }}">
     {{ end }}
     <link rel="stylesheet" href="{{ path.Join "_" (index .Site.Data "rev-manifest" "css/site.css") | relURL }}">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
```

Comment:
We shouldn't use a library hosted by an external entity. Specially if that external entity is known for invasively tracking users.

Either use other existing library or add it to our dependencies and have it on our host.